docs/spec.md

```diff
@@ -235,6 +302,101 @@ Informs the client that channels are no longer available.
 | 4               | uint32  | channel id      |
 | remaining bytes | uint8[] | message payload |
 
+### Get Parameters
+
+Request one or more parameters. Only supported if the server previously declared that it has the `parameters` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"getParameters"`
+- `parameterNames`: string[], leave empty to retrieve all currently set parameters
+- `includeMetaInfo`: boolean, set to true to retrieve additional meta information for each parameter
+<!-- - `subscribe`: boolean, set to true to subscribe to parameter updates -->
+
+#### Example
+
+```json
+{
+  "op": "getParameters",
+  "parameterNames": [
+    "/int_param",
+    "/float_param",
+    "/string_param",
+    "/node/nested_ints_param"
+  ],
+  "includeMetaInfo": true
+}
+```
+
+### Set Parameters
+
+Set one or more parameters. Only supported if the server previously declared that it has the `parameters` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"setParameters"`
+- `parameters`: array of:
+  - `name`: string
+  - `value`: number | boolean | string | number[] | boolean[] | string[]
+
+#### Example
+
+```json
+{
+  "op": "setParameters",
+  "parameters": [
+    { "name": "/int_param", "value": 3 },
+    { "name": "/float_param", "value": 4.1 }
+  ]
+}
+```
+
+### Subscribe Parameter Update
+
+Subscribe to parameter updates. Only supported if the server previously declared that it has the `parameters` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"subscribeParameterUdpates"`
+- `parameterNames`: string[], leave empty to subscribe to all currently known parameters
```

Comment:
Is this meant to replace the existing set of subscriptions or append/merge?

Reply:
It is meant as append/merge. You will be able to subscribe to a single parameter at max once. Will add a comment about that